### PR TITLE
feat(xslt): bump default xslt-version to 3.0

### PR DESCRIPTION
**Pull request by AirQuick**
Following #905, this pull request bumps the default `/x:description/@xslt-version` to `3.0`.

### Further work
Update Wiki

### CODE REVIEW

File: test/xspec-eqname.xspec [hunk 1]
```diff
@@ -114,7 +113,8 @@
 			select="/Q{}variable-child"><variable-child /></x:variable>
 		<!-- The following definition illustrates redefining a variable and mixing notations
 			for its name. The file xspec-variable.xspec would be a good place to test that,
-			except that that file does not use xslt-version="3.0". -->
+			except that that file did not use xslt-version="3.0".
+			TODO: The default @xslt-version has been changed. Consider relocating the test. -->
```
Comment (galtm) [inline test/xspec-eqname.xspec:117]: @AirQuick , the original comment was mine. Would you like me to take care of the TODO item on this line and in `xspec-variable.xspec`, after you merge this pull request?

Comment (AirQuick) [reply test/xspec-eqname.xspec:117]: Yes, much appreciated!

Comment (galtm) [reply test/xspec-eqname.xspec:117]: Will do. I created issue #927 for myself.

**Comment by AirQuick** [comment]
> ### Further work
> Update Wiki

[Done](https://github.com/xspec/xspec/wiki/Writing-Scenarios/_compare/d058f79e812e3a968f3d13ad6a5db4dff3485b79...4c5bd4256c41e8aa6022803c49d897f329473b03)

**Pull request merged**